src/lib/pickles/pickles.ml

```diff
@@ -1733,9 +1733,11 @@ module Make_str (_ : Wire_types.Concrete) = struct
                             ~f:(fun { Impls.Wrap.Proof_inputs.auxiliary_inputs
                                     ; public_inputs
                                     } () ->
+                              (* TODO(dw) pas runtime tables information *)
```

Comment:
En Anglais, on dise 'not'. Ou 'pass' si une erreur :)

Reply:
It was a typo :). Fixed in 9339d908b0c81b84175357449f9af2744cc3ed82.